lib/units/unit.dart

```diff
@@ -167,7 +251,6 @@ abstract class Unit extends SpriteAnimationGroupComponent<UnitAnimationState>
       width++;
       height++;
     }
-    //return false;
```

Comment:
It could probably just be `void` instead, I had bool if it would be impossible for the unit to get a spot, but I don't think we have to worry about that.